backend/app/api/endpoints/admin/device_monitor.py

```
@@ -245,43 +379,283 @@ async def get_device_stats(
+        device_id: Device unique identifier
+        request: Upgrade request with user_id and options
+        db: Database session
+        current_user: Must be admin
+
+    Returns:
+        AdminDeviceActionResponse indicating success/failure
+    """
+    user_id = request.user_id
+
+    # Validate device exists and is online
+    device_kind, online_info = await _get_device_for_action(db, device_id, user_id)
```

Ruff (0.15.6)

[warning] Unpacked variable `device_kind` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)